src/main/templates/facilities.hbs

```diff
@@ -7,21 +7,19 @@
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
     integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
   <title>ISDCF Facilities List</title>
+  <script type="application/javascript" src="facilities.js" defer></script>
+  <style> table td:first-child { width: 9px }</style>
 </head>
 
 <body>
   <nav class="navbar navbar-expand-lg navbar-light bg-light justify-content-between d-print-block">
     <a class="navbar-brand" href="#">ISDCF Facilities List</a>
-    <ul class="navbar-nav d-print-none">
-      <li class="nav-item">
-        <a class="nav-link" href="https://github.com/isdcf/registries">Get the code</a>
```

Comment:
Where would the link to the github repo be -- until we have a proper service setup? #5 suggests opening a new window.